src/utils/tests/patches.test.js

```diff
@@ -436,3 +456,45 @@ describe('utils.js tests', () => {
     comparePatch(patch[0], '/spec', { running: false });
   });
 });
+
+describe('getDeviceBootOrderPatch() tests', () => {
+  const disks = [
+    { disk: {}, name: 'disk-one', bootOrder: 1 },
+    { disk: {}, name: 'disk-two', bootOrder: 2 },
+    { disk: {}, name: 'disk-three', bootOrder: 3 },
+    { disk: {}, name: 'disk-four', bootOrder: 7 },
+  ];
+
+  const interfaces = [
+    { bridge: {}, name: 'nic-one', bootOrder: 4 },
+    { bridge: {}, name: 'nic-two', bootOrder: 5 },
+    { bridge: {}, name: 'nic-three', bootOrder: 6 },
+  ];
+
+  let vm;
+
+  beforeEach(() => {
+    vm = cloneDeep(getVM(false));
```

Comment:
doesn't this function already return brand new object?

Reply:
It does... lol. Fixed!